jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -336,9 +338,15 @@ protected ServletHandler newServletHandler()
     @Override
     protected void startContext() throws Exception
     {
-        ServletContainerInitializerCaller sciBean = getBean(ServletContainerInitializerCaller.class);
-        if (sciBean != null)
-            sciBean.start();
+        for (ServletContainerInitializerCaller  sci : getBeans(ServletContainerInitializerCaller.class))
+        {
+            if (sci.isStopped())
+            {
+                sci.start();
+                if (isAuto(sci))
+                    manage(sci);
+            }
+        }
```

Comment:
This allows for multiple SCI Callers.  It also correctly sets them as managed if this handler starts them.